packages/core/src/Calendar/CalendarView.vue

```
@@ -0,0 +1,62 @@
+  /** The unit the cells inside this view render. */
+  unit: ComputedRef<CalendarUnit>
+}
+
+export const [injectCalendarViewContext, provideCalendarViewContext]
+  = createContext<CalendarViewContext>('CalendarView')
+</script>
+
+<script setup lang="ts">
+import { computed } from 'vue'
+import { Primitive } from '@/Primitive'
+import { useForwardExpose } from '@/shared'
```

ESLint

[error] '`@/shared`' imported multiple times.

(import/no-duplicates)